src/types/label.js

```diff
@@ -127,7 +127,7 @@ function calculatePosition(start, size, adjust = 0, position) {
 
 function drawCallout(ctx, element) {
   const {pointX, pointY, calloutPosition, options} = element;
-  if (element.inRange(pointX, pointY)) {
+  if (element.inRange(pointX, pointY) || !calloutPosition) {
```

Comment:
Turn the checks around, `!calloutPosition` is a faster test and the inRange test is skipped when that is true.

Reply:
+ add a test

Reply:
YES!

Reply:
done